src/app/3d/qgs3dnavigationwidget.cpp

```diff
@@ -196,11 +235,41 @@ Qgs3DNavigationWidget::Qgs3DNavigationWidget( Qgs3DMapCanvas *parent ) : QWidget
   gridLayout->addWidget( mMoveRightButton, 1, 3, 2, 1, Qt::AlignCenter );
   gridLayout->addWidget( mMoveDownButton, 3, 1, 1, 2, Qt::AlignCenter );
   gridLayout->addWidget( mMoveLeftButton, 1, 0, 2, 1, Qt::AlignCenter );
+
+  QHBoxLayout *layout = new QHBoxLayout;
+  layout->addWidget( mCameraInfo );
+  mCameraInfo->setVisible( false );
+
+  QCheckBox *cameraInfoCheckBox = new QCheckBox( this );
+  cameraInfoCheckBox->setText( "Show camera info" );
```

Comment:
The text should be wrapped by tr(). And maybe add "(for debugging)" to make it clear this is not something for ordinary user :)